file_organizer_v2/src/file_organizer/models/audio_transcriber.py

```
@@ -0,0 +1,417 @@
+"""Core audio transcription engine using faster-whisper."""
+
+import time
+from dataclasses import dataclass, field
+from enum import Enum
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional, Union
```

Ruff (0.14.13)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)